modules/mid_registrar/save.c

```diff
@@ -539,10 +539,7 @@ void overwrite_contact_expirations(struct sip_msg *req, struct mid_reg_info *mri
 	for (c = get_first_contact(req); c; c = get_next_contact(c)) {
 		calc_contact_expires(req, c->expires, &e, 1);
 		calc_ob_contact_expires(req, c->expires, &expiry_tick, mri->expires_out);
-		if (expiry_tick == 0)
-			new_expires = 0;
-		else
-			new_expires = expiry_tick - get_act_time();
+		new_expires = expiry_tick == 0 ? 0 : expiry_tick - get_act_time();
```

Comment:
Good idea to normalize the coding style (I approve!), although it is unrelated to the topic of the PR.  On small PRs, it is OK to do one or two of these, however on large PRs, having lots of such changes will make the reviewing more difficult.

Reply:
yes, I should make 2 commits.